frontend/model/state.js

```diff
@@ -426,6 +428,7 @@ const actions = {
     for (const contractID in state.contracts) {
       commit('removeContract', contractID)
     }
+    captureLogsPause()
```

Comment:
Take advantage of the `Vue.nextTick` below and add it there at the very end, this will help ensure that if any problems occur in things like `commit('logout')` they will be caught in the logs.